lab1/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="ISO-8859-1">
+<link rel="stylesheet" href="app.css">
+
+<title>Incremental Game</title>
+</head>
+<body background="theme.jpg">
```

Comment:
Suggestion, if you want to have a background, use CSS to add background rather than using attribute. This way HTML will only be content and CSS for styling.